modules/bullet/space_bullet.cpp

```diff
@@ -41,6 +41,7 @@
 #include "rigid_body_bullet.h"
 #include "servers/physics_server_3d.h"
 #include "soft_body_bullet.h"
+#include <algorithm>
 
```

Comment:
Again, I thought we weren't supposed to use the Standard Template Library; except for a few specified exceptions.

Reply:
Removed `std::vector` in favour of the recently added `LocalVector`.